App/CrisisCleanup/Sources/Feature/Cases/UI/CasesView.swift

```diff
@@ -1,12 +1,46 @@
 import SwiftUI
+import SVGView
 
 struct CasesView: View {
     @Environment(\.translator) var t: KeyAssetTranslator
 
     @ObservedObject var viewModel: CasesViewModel
+    let authenticateViewBuilder: AuthenticateViewBuilder
     let incidentSelectViewBuilder: IncidentSelectViewBuilder
+    let incidentsTypeIconsAssetsFolder = "incident_type_icons/"
 
-    var body: some View {
-        Text(t("casesVue.cases"))
+    @State var showIncidentSelect = false
+
+    var body: some View{
+        VStack {
+            HStack {
+                Button {
+                    showIncidentSelect.toggle()
+                } label: {
+                    HStack{
+                        Image(incidentsTypeIconsAssetsFolder+viewModel.incidentsData.selected.disasterLiteral, bundle: .module).foregroundColor(Color.blue)
+                    }
+                    let selectedIncident = viewModel.incidentsData.selected.shortName
+                    let title = selectedIncident.isEmpty
+                    ? t.translate(
+                        TopLevelDestination.menu.titleTranslateKey,
+                        TopLevelDestination.menu.titleLocalizationKey
+                    )
+                    : selectedIncident
+                    Text(title)
```

Comment:
The Cases/Work screen does not display the title of the Incident like the Menu screen does. Remove this text